### DISCUSSION TITLE
Whisky + gptk possibly usable for autodesk application

### DISCUSSION THREAD

**Discussion by AliG1030** [Whisky-App/Whisky#776] [General]
Hello guys, first of all, a huge thanks to the community. All the works & research put in here that has for sure contributed to my recent upgrade of my mac, allowing me to play games more than ever. But on the other side, i'm also using my computer as a work device. I have a licence for autodesk product, i do precise this because i'm not here to get some help for hakhing softwares. 

Also i have to precise that i'm not an expert in coding ( that's why i do thank a lot the community cause you helped me a lot throught the process of installing whisky and gptk ).

Anyway i was wondering, i work with autocad on mac but regarding revit i'm using it with parallel. I tried multiple times to install it through whisky ( i even tried with crossover ). Crossover seemed more friendly with the installing process but it does block at some point. 

So my question is, do you think that using revit with bottles like whisky or crossover is something doable at the moment and i'm just doing it wrong in the process ? 

Or those software at the moment are just made for games and not software ? 

My questions might not seem relevant and do lack of informations regarding the way i tried to install it, i'll add more infos about that once some of you will think that deserves reflexions.

Thanks a lot for your time and consideration

**Comment by jamalkhaffaf** [comment]
tried to install it but it fails because "a file is missing during download". figured its not compatible yet? possible future work?
